docs/csharp/programming-guide/interop/how-to-use-platform-invoke-to-play-a-wave-file.md

```diff
@@ -9,9 +9,9 @@ helpviewer_keywords:
   - ".wav files"
 ms.assetid: f7f62f53-e026-4c40-b221-3a26adb0c2c5
 ---
-# How to: Use Platform Invoke to Play a Wave File (C# Programming Guide)
+# How to use platform invoke to play a WAV file (C# Programming Guide)
```

Comment:
I replaced Wave with WAV. I think it's more common and less confusing to say WAV rather than Wave.